.github/workflows/version-checks.yaml

```diff
@@ -0,0 +1,57 @@
+name: Version Consistency Checks
+
+on:
+  push:
+    branches-ignore:
+      - "gh-readonly-queue/**"
+  pull_request:
+  merge_group:
+  schedule: # Trigger a job on default branch at 4AM PST everyday
```

Comment:
I don't think this check needs to run on schedule if it's already run on pushes and PRs.